admin_webapp/templates/endorsement/request_detail.html

```diff
@@ -0,0 +1,46 @@
+{%- extends "base.html" -%}
+
+{% macro yn( value ) -%}
+{%- if value -%}
+<span class='badge bg-success'>Y</span>
+{%- else -%}
+<span class='badge bg-warning'>N</span>
+{%- endif -%}
+{%- endmacro %}
+
+{% macro flipbtn( url ) %}
+<form class="d-inline" action="{{url}}" method="post">
+  <button class="btn btn-primary btn-sm" type="submit" name="flip" value="flip">flip</button>
+  <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+</form>
+{%- endmacro %}
+
+{%block content%}
+<h1>Endorsement Request</h1>
+<div class="card">
+<table>
+<tr><td>Category:</td><td> {{endo_req.archive}}.{{endo_req.subject_class}}</td></tr>
+<tr><td>Endorser:</td><td> {% if endo_req.endorsement %}<a href='{{url_for('user.detail', user_id=endo_req.endorsement.endorser_id)}}'>
+    {{endo_req.endorsement.endorser.nickname}}</a>{% else %}(endorsement missing){% endif %}</td></tr>
+<tr><td>Endorsee:</td><td> <a href='{{url_for('user.detail', user_id=endo_req.endorsee_id)}}'>
+      {{endo_req.endorsee.nickname}}</a></td></tr>
+
+<tr><td>Valid?:</td><td> {{ yn(endo_req.flag_valid) }}  {{flipbtn(url_for('endorsement.flip_valid', endorsement_req_id=endo_req.request_id))}}</td></tr>
+
+<tr><td>Positive?:</td><td> {{ yn( endo_req.endorsement.point_value)}} {{ flipbtn(url_for('endorsement.flip_score', endorsement_req_id=endo_req.request_id))}} </td></tr>
+
+<tr><td>Knows Personally?:</td><td> {{ yn( endo_req.audit.flag_knows_personally ) }}</td></tr>
+<tr><td>  Has Seen Paper?:</td><td> {{ yn(endo_req.audit.flag_seen_paper ) }}</td></tr>
+<tr><td>  Type:</td><td> {% if endo_req.endorsement %}{{endo_req.endorsement.type}}{% else %}(endorsement missing){% endif %}</td></tr>
+<tr><td>Points:</td><td> {{endo_req.point_value}}</td></tr>
+<tr><td>Request Id:</td><td> {{endo_req.request_id}}</td></tr>
+<tr><td>Endorsement:</td><td> {% if endo_req.endorsement %}<a href='{{url_for('endorsement.detail', endorsement_id=endo_req.endorsement.endorsement_id)}}'>{{endo_req.endorsement.endorsement_id}}</a>{% else %}(endorsement missing){% endif %}</td></tr>
+<tr><td>Issued When:</td><td> {{endo_req.issued_when}}</td></tr>
+<tr><td>Session Id:</td><td> {% if endo_req.audit %}<a href='{{url_for('session.detail', session_id=endo_req.audit.session_id)}}'>{{endo_req.audit.session_id}}</a>{% else %}(session missing){% endif %}</td></tr>
+<tr><td>Remote Hostname:</td><td> none</td></tr>
+<tr><td>Remote Address:</td><td> {{endo_req.audit.remote_addr}}</td></tr>
+<tr><td>Tracking Cookie:</td><td> {{endo_req.audit.tracking_cookie}}</td></tr>
+<tr><td>Comment:</td><td> {{endo_req.audit.comment}}none</td></tr>
```

Comment:
1 question and 1 request for this line.
Question: Will it always display "none" or is that just if the value is null?

Request: Can we add bold or emphasis here so that it pops out a bit at the end of the table? This is fine to have at the end as there could be a lengthy comment, but we want that to pop out a bit if we go to review the endorsement.

Reply:
> 1 question and 1 request for this line. Question: Will it always display "none" or is that just if the value is null?

This seems to be empty in the DB these days. We can add a hostname lookup once we do it in python. I'll set it to display.
 
> Request: Can we add bold or emphasis here so that it pops out a bit at the end of the table? This is fine to have at the end as there could be a lengthy comment, but we want that to pop out a bit if we go to review the endorsement.

Bold is no problem. :+1:

Reply:
@jimentwood I've split the page into two cards to represent the fields related to the endorsement request distinct from the fields related to the endorsement. 
![Selection_069](https://user-images.githubusercontent.com/1616/195438320-0d10b3ec-053f-4688-9334-05a4ca531428.png)

I've made the comment bold and added the hostname.

Reply:
I did this because half of the fields were about the initial request to get an endorsement and the other half were about the endorser's reponse.

Reply:
I'm glad you like it.